XamlControlsGallery/ControlPages/MenuBarPage.xaml.cs

```diff
@@ -26,5 +26,23 @@ public MenuBarPage()
         {
             this.InitializeComponent();
         }
+
+        private void OnElementClicked(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+            var selectedFlyoutItem = sender as MenuFlyoutItem;
+            string exampleNumber = selectedFlyoutItem.Name.Substring(0, 1);
```

Comment:
An alternative approach to using the first letter in the Name could be to use the Tag property to pass information about which TxtBlock to update.

Reply:
Or set the Tag of the parent MenuBar and bind the TextBlock's text to that Tag. Then we wouldn't need to name everything.